resources/policies/rego-block-image-policy.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  annotations:
+    io.kubewarden.policy.category: software-supply-chain
+    io.kubewarden.policy.severity: high
+  name: containers-block-specific-image-names
+spec:
+  module: registry://ghcr.io/kubewarden/policies/containers-block-specific-image-names:v1.0.1
```

Comment:
Can we use last released version of this policy instead of hardcoded 1.0.1?
I don't see `latest` tag on rego-policy-library, but our other policies have it.

Reply:
There is no `latest` tag indeed. I checked before.

Reply:
As discussed during the daily, we will keep the hardcoded policy.